app/assets/stylesheets/templates/_step.scss

```diff
@@ -6,4 +6,15 @@
     content: "";
     display: block;
   }
+
+  .form-card {
+    max-width: 72rem;
+
+    @include media($tablet-up) {
+      margin: {
+        left: auto;
+        right: auto;
+      }
+    }
+  }
```

Comment:
What do you think about making this a more generic class that we can then apply to more elements than just the `.form-card`? I say this because I know we'll need it for the navigation, and eventually the flash messages